OpenSim/Simulation/Model/Frame.h

```diff
@@ -68,88 +87,135 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     Frame();
 
     virtual ~Frame() {};
+
     /** @name Spatial Operations for Frames
     These methods allow access to the frame's transform and some convenient
     operations that could be performed with this transform.*/
     /**@{**/
 
     /**
-    * Get the transform that describes the translation and rotation of this
-    * frame (F frame) relative to the ground frame (G frame).  This method
-    * returns the transform converting quantities expressed in F frame to
-    * quantities expressed in the G frame. This is mathematically stated as,
-    * vec_G = X_GF*vec_F ,
-    * where X_GF is the transform returned by getGroundTransform.
-    *
-    * @param state       The state applied to the model when determining the
-    *                    transform.
-    * @return transform  The transform between this frame and the ground frame
+    Get the transform of this frame (F) relative to the ground frame (G).
+    It transforms quantities expressed in F into quantities expressed
+    in G. This is mathematically stated as:
+        vec_G = X_GF*vec_F ,
+    where X_GF is the transform returned by getGroundTransform.
+
+    @param state       The state applied to the model when determining the
+                       transform.
+    @return transform  The transform between this frame and the ground frame
     */
-    SimTK::Transform getGroundTransform(const SimTK::State& state) const {
-        return calcGroundTransform(state);
-    }
+    const SimTK::Transform& getGroundTransform(const SimTK::State& state) const;
 
     /**
-    * Get the transform that describes the translation and rotation of this
-    * frame (F frame) relative to another frame (A frame).  This method returns
-    * the transform converting quantities expressed in F frame to quantities
-    * expressed in the A frame. This is mathematically stated as,
-    * vec_A = X_AF*vec_F ,
-    * where X_AF is the transform returned by this method.
-    *
-    * @param state       The state applied to the model when determining the
-    *                    transform.
-    * @param otherFrame  a second frame
-    * @return transform  The transform between this frame and otherFrame
+    Find the transform that describes this frame (F) relative to another
+    frame (A). It transforms quantities expressed in F to quantities expressed
+    in A. This is mathematically stated as:
+    This is mathematically stated as:
+        vec_A = X_AF*vec_F ,
```

Comment:
Since this is a transform and includes a translational shift, this should use a point or station like `pt_A = X_AF*pt_F` rather than a plain `vec` which may represent just a direction. It might help to clarify if the comment also shows rotating a vector, like `vec_A = X_AF.R()*vec_F` so people don't shift by accident.